src/minecraft/MinecraftManager.js

```diff
@@ -36,11 +36,12 @@ class MinecraftManager extends CommunicationBridge {
     })
   }
 
-  onBroadcast({ username, message }) {
+  onBroadcast({ username, message, replyingTo }) {
     console.log(chalk.blue(`Minecraft Broadcast > ${username}: ${message}`))
 
     if (this.bot.player !== undefined) {
-      this.bot.chat(`/gc ${username}: ${message}`)
+      console.log(`/gc ${replyingTo ? `Replying to ${replyingTo}:` : `${username}:`} ${message}`)
+      this.bot.chat(`/gc ${replyingTo ? `Replying to ${replyingTo}:` : `${username}:`} ${message}`)
```

Comment:
The author's username isn't include in the message when sending a reference message, it's simply saying `"Repling to <other-username>: <message>"` instead of `"<username>: <message>"`, the format could be changed to:

    <username> replying to <other-username>: <message>

Would include who is actually sending the message in this case.

Reply:
on another note, the console log statement can be removed too, we're already sending Minecraft broadcast messages on line 40, we don't need another one.